generate-devsite.sh

```diff
@@ -44,6 +44,16 @@ sed -i -e 's/^    //' ./yaml/toc.yml
 sed -i -e '/name: I[A-Z]/{N;d;}' ./yaml/toc.yml
 sed -i -e '/^ *\@google-cloud.*:interface/d' ./yaml/toc.yml
 
+echo "deleting lines with :interface"
+# Delete uids with :interface and line before
+# See https://www.theunixschool.com/2012/06/sed-25-examples-to-delete-line-or.html
+sed -i -n -e '/:interface/{s/.*//;x;d;};x;p;${x;p;}' ./yaml/toc.yml
```

Comment:
When do we do something here rather than in the rushstack fork? Could we just not include these in the TOC?

Also, does this mean the user could get to a page without an entry in the TOC?

Reply:
Anything in Rushstack makes rebasing with origin a pain. I'll try to keep as much out of it as possible.

Reply:
Yes, Interfaces wouldn't have an entry on the left. It's a trade off between a usable left nav and having _everything_ in the left nav. Here's this PR deployed: 

https://cloud.google.com/nodejs/docs/reference/storage/latest/overview#interfaces